Getting `go-header_bad.go:1: missing error "Expected:TITLE\\., Actual: TITLE!"` when running make

### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).
- [X] Yes, I've tried with the standalone linter if available. (https://golangci-lint.run/usage/linters/)

### Description of the problem

I am trying to make the library from source. I clone the repo and run `make` and I get 

```
    linters_test.go:37: 
        	Error Trace:	linters_test.go:37
        	            				linters_test.go:216
        	            				linters_test.go:57
        	Error:      	Received unexpected error:
        	            	go-header_bad.go:1: missing error "Expected:TITLE\\., Actual: TITLE!"
        	Test:       	TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
```

This fails the build. I am running this on an Intel Macbook

### Version of golangci-lint

<details>
Built from source
</details>

### Configuration file

<details>
n/a
</details>

### Go environment

<details>

```console
$ go version && go env
# go version devel go1.18-b37a5391f9 Tue Dec 7 20:14:56 2021 +0000 darwin/amd64
```

</details>

### Verbose output of running

<details>

n/a

</details>

### Code example or link to a public repository

<details>

n/a

</details>

I have the same problem locally:

```console
$ T=go-header_bad.go make test_linters
GL_TEST_RUN=1 go test -v ./test -count 1 -run TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
=== RUN   TestSourcesFromTestdataWithIssuesDir
    linters_test.go:41: testdata/*.go
=== RUN   TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
=== PAUSE TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
=== CONT  TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
    linters_test.go:216: [run --allow-parallel-runners --disable-all --print-issued-lines=false --out-format=line-number --max-same-issues=100 -Egoheader -c testdata/configs/go-header.yml testdata/go-header_bad.go]
    linters_test.go:37: 
                Error Trace:    linters_test.go:37
                                                        linters_test.go:217
                                                        linters_test.go:57
                Error:          Received unexpected error:
                                go-header_bad.go:1: missing error "Expected:TITLE\\., Actual: TITLE!"
                Test:           TestSourcesFromTestdataWithIssuesDir/go-header_bad.go
--- FAIL: TestSourcesFromTestdataWithIssuesDir (0.34s)
    --- FAIL: TestSourcesFromTestdataWithIssuesDir/go-header_bad.go (0.60s)
FAIL
FAIL    github.com/golangci/golangci-lint/test  1.352s
FAIL
make: *** [Makefile:43: test_linters] Error 1
```

--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,7 @@ require (
 	github.com/butuzov/ireturn v0.1.1
 	github.com/charithe/durationcheck v0.0.9
 	github.com/daixiang0/gci v0.3.1
-	github.com/denis-tingajkin/go-header v0.4.2
+	github.com/denis-tingaikin/go-header v0.4.3
 	github.com/esimonov/ifshort v1.0.4
 	github.com/fatih/color v1.13.0
 	github.com/fzipp/gocyclo v0.4.0

--- a/pkg/golinters/goheader.go
+++ b/pkg/golinters/goheader.go
@@ -4,7 +4,7 @@ import (
 	"go/token"
 	"sync"
 
-	goheader "github.com/denis-tingajkin/go-header"
+	goheader "github.com/denis-tingaikin/go-header"
 	"golang.org/x/tools/go/analysis"
 
 	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"

--- a/pkg/lint/lintersdb/manager.go
+++ b/pkg/lint/lintersdb/manager.go
@@ -362,7 +362,7 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		linter.NewConfig(golinters.NewGoHeader()).
 			WithSince("v1.28.0").
 			WithPresets(linter.PresetStyle).
-			WithURL("https://github.com/denis-tingajkin/go-header"),
+			WithURL("https://github.com/denis-tingaikin/go-header"),
 
 		linter.NewConfig(golinters.NewGoimports()).
 			WithSince("v1.20.0").

--- a/test/testdata/go-header_bad.go
+++ b/test/testdata/go-header_bad.go
@@ -1,5 +1,5 @@
 /*MY TITLE!*/ // ERROR `Expected:TITLE\., Actual: TITLE!`
 
-//args: -Egoheader
-//config_path: testdata/configs/go-header.yml
+// args: -Egoheader
+// config_path: testdata/configs/go-header.yml
 package testdata

--- a/test/testdata/go-header_good.go
+++ b/test/testdata/go-header_good.go
@@ -1,5 +1,5 @@
 /*MY TITLE.*/
 
-//args: -Egoheader
-//config_path: testdata/configs/go-header.yml
+// args: -Egoheader
+// config_path: testdata/configs/go-header.yml
 package testdata

